lua/autorun/client/TrueCulling.lua

```diff
@@ -0,0 +1,68 @@
+jit.on()
+
+CULLFRONT = math.sin(0.57357644) -- Degrees forward angle in sine
+
+function main()
+	allEnts = ents.GetAll()
+	if engine.TickCount()%4 == 0 then
+		for a, ents_ in pairs(allEnts) do
+			if engine.TickCount()%10 == 0 then
```

Comment:
So this "performance optimization" addon computes the static value `engine.TickCount()%10 == 0` 4 times for each entity? Do you know what a variable is?

Reply:
Also you previously already checked `engine.TickCount()%4==0`. So is this supposed to run every forth tick, or every tenth? Also you should use FrameNumber() because Think runs every frame (client), not every tick (server).